content/synchronization_model.tex

```diff
@@ -19,9 +19,9 @@
 \begin{tabular}{p{0.2\textwidth} | p{0.7\textwidth}}
 {}
 &
-{ Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
+Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
```

Comment:
@manjugv I think a local (i.e. a thread) may update the variable with an AMO/put